tests/pf/test_forward_Mag_Linear.py

```diff
@@ -1,77 +1,220 @@
-import unittest
 import discretize
```

Comment:
We should use these imports and update the type hints so they can work on Python 3.8
```suggestion
from typing import List, Tuple
import discretize
```